Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.js

```diff
@@ -694,6 +702,108 @@ WI.DOMTreeContentView = class DOMTreeContentView extends WI.ContentView
+            this._userPreferencesOverridesPopover.present(calculateTargetFrame(), preferredEdges);
+        };
+
+        let accessibilityHeader = contentElement.appendChild(document.createElement("h5"));
+        accessibilityHeader.textContent = WI.UIString("Accessibility Preferences", "Accessibility Preferences @ User Preferences Overrides", "Header for section with accessibility user preferences.");
+
+        const userPreferenceValues = [InspectorBackend.Enum.Page.UserPreferenceValue.On, InspectorBackend.Enum.Page.UserPreferenceValue.Off];
+
+        // COMPATIBILITY (macOS 13.0, iOS 16.0): `PrefersReducedMotion` value for `Page.UserPreferenceName` did not exist yet.
+        if (InspectorBackend.Enum.Page?.UserPreferenceName.PrefersReducedMotion)
+            createSelectElement(WI.UIString("Reduce motion", "Reduce motion @ User Preferences Overrides", "Label for input to override the preference for reduced motion."), InspectorBackend.Enum.Page.UserPreferenceName.PrefersReducedMotion, userPreferenceValues, WI.CSSManager.UserPreferenceDefaultValue);
+
+        // COMPATIBILITY (macOS 13.0, iOS 16.0): `PrefersContrast` value for `Page.UserPreferenceName` did not exist yet.
+        if (InspectorBackend.Enum.Page?.UserPreferenceName.PrefersContrast)
+            createSelectElement(WI.UIString("Increase contrast", "Increase contrast @ User Preferences Overrides", "Label for input to override the preference for high contrast."), InspectorBackend.Enum.Page.UserPreferenceName.PrefersContrast, userPreferenceValues, WI.CSSManager.UserPreferenceDefaultValue);
+
+        this._userPreferencesOverridesPopover.presentNewContentWithFrame(contentElement, calculateTargetFrame(), preferredEdges);
+    }
+
+    _userPreferenceValueLabel(preferenceValue, systemValue)
+    {
+        switch (preferenceValue) {
+        case WI.CSSManager.UserPreferenceDefaultValue:
+            return WI.UIString("System (%s)", "System @ User Preferences Overrides", "Label for a preference that matches the default system value. The system value is shown in parentheses.").format(systemValue);
+        case InspectorBackend.Enum.Page.UserPreferenceValue.On:
+            return WI.UIString("On", "On @ User Preferences Overrides", "Label for a preference that is turned on.");
+        case InspectorBackend.Enum.Page.UserPreferenceValue.Off:
+            return WI.UIString("Off", "Off @ User Preferences Overrides", "Label for a preference that is turned off.");
+        }
+
+        console.assert(false, "Unknown user preference");
+        return "";
+    }
+
+    _defaultUserPreferencesDidChange()
+    {
+        if (!this._userPreferencesOverridesPopover?.element)
+            return;
+
+        for (let [preferenceName, systemValue] of WI.cssManager.defaultUserPreferences) {
+            let defaultOptionElement = this._userPreferencesOverridesPopover.element.querySelector(`select[name=${preferenceName}] > option[value=${WI.CSSManager.UserPreferenceDefaultValue}]`);
```

Comment:
I'm skeptical of this approach to finding the select elements to update them... Could we consider instead of a subclass of `WI.Popover` (an example being `WI.CSSDocumentationPopover`) that keeps a more structured understand of the elements for the preferences it is showing?

Reply:
Moved to a new `OverrideUserPreferencesPopover` that subclasses `Popover` and holding an internal map of elements to change.